x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_alerts_query.ts

```diff
@@ -23,7 +23,7 @@ export const useAlertsQueryImpl = () => {
   const getAlertsEsQuery = useCallback(
     (status?: AlertStatus) =>
       createAlertsEsQuery({ dateRange: searchCriteria.dateRange, hostNodeNames, status }),
-    [hostNodeNames, searchCriteria.dateRange]
+    [hostNodeNames, searchCriteria?.dateRange]
```

Comment:
Will the `searchCriteria` ever be undefined?

Reply:
oh, this seems to be a leftover while I was trying something.